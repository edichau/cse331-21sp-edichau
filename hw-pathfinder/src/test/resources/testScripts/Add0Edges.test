CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B