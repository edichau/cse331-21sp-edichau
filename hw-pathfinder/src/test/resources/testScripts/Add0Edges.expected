created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph